core/src/test/kotlin/org/evomaster/core/search/gene/sql/geometric/SqlMultiPolygonGeneTest.kt

```diff
@@ -0,0 +1,114 @@
+package org.evomaster.core.search.gene.sql.geometric
+
+import org.evomaster.client.java.controller.api.dto.database.schema.DatabaseType
+import org.evomaster.core.search.gene.FloatGene
+import org.evomaster.core.search.service.Randomness
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class SqlMultiPolygonGeneTest {
+
+    val rand = Randomness()
```

Comment:
as in previous comment: make sure to set a seed (can be the same) in each test, to avoid non-determinism and dependencies between tests

Reply:
ok